<!DOCTYPE html>
<html>

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  <title>Retracing Prenatal Testing Algorithms</title>
  <meta name="description" content="Abstract">

  <link rel="stylesheet" href="/blog/css/main.css">
  <link rel="canonical" href="https://mhoehle.github.io/blog/2018/06/14/prc.html">
  <link rel="alternate" type="application/rss+xml" title="Theory meets practice..." href="https://mhoehle.github.io/blog/feed.xml">
</head>

<!-- https://docs.mathjax.org/en/v2.7-latest/start.html -->
<script type="text/javascript" async
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML">
</script>




  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/blog/">Theory meets practice...</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
        
          
          <a class="page-link" href="/blog/about/">About</a>
          
        
          
        
          
        
          
        
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <article class="post" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title" itemprop="name headline">Retracing Prenatal Testing Algorithms</h1>
    <p class="post-meta"><time datetime="2018-06-14T00:00:00+02:00" itemprop="datePublished">Jun 14, 2018</time></p>
  </header>

  <div class="post-content" itemprop="articleBody">
    <h2 id="abstract">Abstract</h2>
<p>A good understanding of the statistical procedure used to calculate
trisomy 21 (Down syndrome) risk in combined first trimester screening is
a precondition for taking an informed decision on how to proceed with
the screening results. For this purpose we implement the Fetal Medicine
Foundation (FMF) Germany procedure described in <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span>. To allow
soon-to-become-parents an insight on what the procedure does and how
sensitive conclusions are to perturbations, we wrote a small Shiny app
as part of the R package <code>trisomy21risk</code> to visualize the
results. We hope the tool can be helpful in the individual decision
making process associated with first trimester screening.</p>
<center>
<img
src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/aa/Feindiagnostik_%28Ultraschall%29.jpg/320px-Feindiagnostik_%28Ultraschall%29.jpg" />
</center>
<p><br>
<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/88x31.png"/></a>
This work is licensed under a <a rel="license"
href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons
Attribution-ShareAlike 4.0 International License</a>. The
markdown+Rknitr source code of this blog is available under a <a
href="https://www.gnu.org/licenses/gpl-3.0.html">GNU General Public
License (GPL v3)</a> license from github.</p>
<h2 id="introduction">Introduction</h2>
<p>Biostatisticians predominantly analyse samples from populations of
dead and diseased in the hope of substantiating answers to question
aimed at preventing the adverse event in future individuals. Less common
is the personalized <span class="math inline">\(n=1\)</span> case, where
-based on historical data- one has to choose an optimal test sequence or
treatment regime for a specific individual. However, when interpreting
your own laboratory values that is what you have to do - implicitly or
explicitly!</p>
<p>In the case of first trimester <a
href="en.wikipedia.org/wiki/Prenatal_testing">prenatal testing</a> for
<a href="https://en.wikipedia.org/wiki/Down_syndrome">Down syndrome</a>,
aka. trisomy 21 (T21), parents are at the end of the screening routine
given a piece of paper containing the results of a combined test. This
consists of</p>
<ul>
<li>the <a href="https://en.wikipedia.org/wiki/Nuchal_scan">nuchal
translucency</a> (NT) measurement (in mm) of the fetus</li>
<li>concentration of the Pregnancy-associated plasma protein A <a
href="https://en.wikipedia.org/wiki/Pregnancy-associated_plasma_protein_A">(PAPP-A)</a>
and the human chorionic gonadotropin <a
href="https://en.wikipedia.org/wiki/Human_chorionic_gonadotropin">(β-HCG)</a>
hormone in the serum of the mother (measured in <a
href="https://en.wikipedia.org/wiki/International_unit">miU</a>/ml)</li>
<li>(optional) results for other markers (e.g. ethnicity, maternal
smoking, fetal heart rate, visibility of the nasal bone on the
ultrasound)</li>
</ul>
<p>From the biomarker values NT, PAPPA-A and β-HCG (and possibly
additional markers) a risk for T21 is calculated and presented as an
odds, say, 1:486. The question for the concerned soon-to-become-parents
is now, whether to proceed with further tests - for example a DNA based
approach in maternal blood plasma, a so called <strong>non-invasive
prenatal test</strong> (NIPT) or, further down the line, an invasive <a
href="https://en.wikipedia.org/wiki/Amniocentesis">amniotic fluid
test</a>. Recommendations for when to perform which additional screening
steps exist and are discussed with the supervising gynecologist, but at
the end its your choice as soon-to-become-parents.</p>
<p>The aim of this blog post is to illustrate how we failed to identify
and understand the algorithm used to calculate the risk. As a
consequence, we attempt a transparent and reproducible risk calculation
algorithm based on the procedure described in <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span>. To enhance risk
communication we wrote an interactive Shiny app to calculate the risk,
perform sensitivity analyses and graph distributions. Even though we are
not experts on prenatal screening and have no access to any data, we
hope such tools can inspire the experts or even be helpful in the
individual decision making process associated with first trimester
screening.</p>
<h3 id="what-is-the-fmf-algorithm-2012">What is the FMF Algorithm
2012?</h3>
<p>The first step in a scientific approach towards the problem is to
understand, how the <span class="math inline">\(y\)</span> in the
reported <span class="math inline">\(1:y\)</span> odds was determined.
For this purpose the exemplary result report (in this case from a German
prenatal diagnosis facility) contained a cryptic note to a “FMF
Algorithm 2012”. Unfortunately, no further reference to any scientific
article, no link to a website with additional information nor any
separate pamphlet was provided. Nevertheless, the soon-to-be-parents are
assured that “The model for calculating the risk is based on intensive
research work coordinated by the fetal medicine foundation (registered
charity 1037116)”. Cranking the google handle we found that there exists
both an UK <a href="https://fetalmedicine.org">Fetal Medicine
Foundation</a> (FMF), which has the stated registered charity number, as
well as a <a
href="https://www.fmf-deutschland.info/en/fmf-deutschland/">Fetal
Medicine Foundation Germany</a>. For reasons unclear to the outsider
these two organizations use different risk algorithms: FMF-Germany uses
a so called DoE-algorithm (DoE = Degree of extremeness) documented in
the <span class="citation" data-cites="merz_etal2008">Merz et al.
(2008)</span>, <span class="citation" data-cites="merz_etal2011">Merz et
al. (2011)</span> and <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span> (open-access)
sequence of papers. In contrast FMF-UK uses software following “The
First Trimester Screening module 2012 algorithm”, which uses so called
MoMs (multiples of the median). What this entails we could only find in
a <a
href="https://doclegend.com/queue/03-guide-for-the-first-trimester-screening-module_5a17d7fbd64ab28f8c88fe0e_pdf?queue_id=5b205c29d64ab23ffc4c2e34">rogue
access software manual</a>: the document contains a large body of
references sketching the different analysis components - in particular
<span class="citation" data-cites="kagan_etal2008a">Kagan, Wright,
Baker, et al. (2008)</span> and <span class="citation"
data-cites="kagan_etal2008b">Kagan, Wright, Valencia, et al.
(2008)</span> seem to cover the essentials components of the algorithm.
However, for each component (nuchal translucency, β-HCG, PAPP-A, nasal
bone, ductus venosus, heart rate, …) further references are given
without it being clear, if or how they enter the algorithm. As an
example, the FMF 2012 algorithm seems to analyze the NT measurement by a
mixture model approach described in <span class="citation"
data-cites="wright_etal2008">Wright et al. (2008)</span> so the original
approach in <span class="citation" data-cites="kagan_etal2008a">Kagan,
Wright, Baker, et al. (2008)</span> now seems obsolete. This might all
be scientifically justified, but for an outsider it is impossible to
retrace what is done: Even though we spent quite some time searching the
FMF-UK website, which provides access to many of the FMF’s 957
scientific publications and watched <a
href="http://www.perkinelmer.co.uk/lab-products-and-services/maternal-fetal-health/FMF-risk-algorithm.html">videos
about the algorithm</a>, we could not find a more coherent document on
what the FMF 2012 algorithm exactly is. Even the FMF <a
href="https://fetalmedicine.org/research/assess/trisomies">on-line risk
calculator</a> does not contain a specific reference and seems to
implement its own a variant of the algorithm (<a
href="https://fetalmedicine.org/assets/b302e11a/js/_fmf.min.js?v=14397">undocumented
javascript source code</a>).</p>
<p>At this point it is instrumental to recall paragraph V.3.3 in the <a
href="https://www.rki.de/DE/Content/Kommissionen/GendiagnostikKommission/Richtlinien/RL-VorgeburtlicheRisikoabklaerung.pdf?__blob=publicationFile">Commission
on Genetic Testing</a>’s guideline for prenatal screening. It demands
that the algorithm for the computation of such risks to be
<em>comprehensible, scientifically justified and published in a peer
review process</em>. In particular the guideline stresses that
<em>scientifically justified</em> means:</p>
<div class="blackbox">
<em>Demanded is publication of the entire algorithm with detailed
software specification, which precisely and comprehensible describes how
the risks are calculated.</em>
</div>
<p>
<p>With such a strong statement by the guideline about the
<strong>transparency</strong> and <strong>reproducibility</strong> of
the algorithm the above hide-and-seek seems absurd.</p>
<p>Since we at the time of testing (wrongly?) believed that the
FMF-Germany algorithm was the one used in the lab report and its papers
appeared comprehensible enough for a re-implementation, we decided to
scrutinize the FMF-Germany algorithm further in order to understand the
computations. Aim was to understand how components influenced the
calculation, perform a sensitivity analysis and better understand the
uncertainty associated with the reported risk. The FMF Algorithm 2012
was expected to be similar in style.</p>
<h2 id="the-fmf-germany-procedure---prc-3.0">The FMF-Germany Procedure -
PRC 3.0</h2>
<p>The procedure described in <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span> entitled
<strong>Prenatal Risk Calculation</strong> (PRC) uses a classic Bayesian
diagnostic approach, where one computes a posterior odds for the fetus
having T21 as</p>
<p><span class="math display">\[
\text{Posterior odds} = \text{Likelihood ratio} \cdot \text{Prior odds}
\]</span></p>
<p>In the context of T21 first trimester screening, the prior odds is
known to depend on maternal age and gestational age (at the time of
screening) of the fetus. Tables for this can be found in, e.g., Table 4
of <span class="citation" data-cites="snijders_etal1999">Snijders et al.
(1999)</span>, which is shown below in the form of odds <span
class="math inline">\(1:x\)</span> for maternal age gaps of 5 years and
selected <a
href="https://en.wikipedia.org/wiki/Gestational_age">gestational
ages</a> (measured in weeks). The gestational weeks relevant for the
first trimester screening are the gestational weeks 10-13. Using a
linear interpolation we obtain values for maternal and gestational age
not covered in the table - see the <code>trisomy21risk</code> package
code for details.</p>
<table class="table table-striped" style="margin-left: auto; margin-right: auto;">
<caption>
Table 1: Background risk in matrix form for T21 for maternal age (in
years) and gestational age (in weeks) by Snijders et al. (1999).
</caption>
<thead>
<tr>
<th style="text-align:right;">
Maternal Age
</th>
<th style="text-align:right;">
W10
</th>
<th style="text-align:right;">
W12
</th>
<th style="text-align:right;">
W14
</th>
<th style="text-align:right;">
W16
</th>
<th style="text-align:right;">
W20
</th>
<th style="text-align:right;">
W40
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
983
</td>
<td style="text-align:right;">
1068
</td>
<td style="text-align:right;">
1140
</td>
<td style="text-align:right;">
1200
</td>
<td style="text-align:right;">
1295
</td>
<td style="text-align:right;">
1527
</td>
</tr>
<tr>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
870
</td>
<td style="text-align:right;">
946
</td>
<td style="text-align:right;">
1009
</td>
<td style="text-align:right;">
1062
</td>
<td style="text-align:right;">
1147
</td>
<td style="text-align:right;">
1352
</td>
</tr>
<tr>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
576
</td>
<td style="text-align:right;">
626
</td>
<td style="text-align:right;">
668
</td>
<td style="text-align:right;">
703
</td>
<td style="text-align:right;">
759
</td>
<td style="text-align:right;">
895
</td>
</tr>
<tr>
<td style="text-align:right;">
35
</td>
<td style="text-align:right;">
229
</td>
<td style="text-align:right;">
249
</td>
<td style="text-align:right;">
266
</td>
<td style="text-align:right;">
280
</td>
<td style="text-align:right;">
302
</td>
<td style="text-align:right;">
356
</td>
</tr>
<tr>
<td style="text-align:right;">
40
</td>
<td style="text-align:right;">
62
</td>
<td style="text-align:right;">
68
</td>
<td style="text-align:right;">
72
</td>
<td style="text-align:right;">
76
</td>
<td style="text-align:right;">
82
</td>
<td style="text-align:right;">
97
</td>
</tr>
<tr>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
18
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
23
</td>
</tr>
</tbody>
</table>
<p>
<p>From the table we see that the background risk increases with
maternal age and decreases with gestational age. The latter is related
to the fact that a T21 fetus has an increased risk of spontaneous
abortion. One criticism of using the maternal age as background risk is
that age is merely a proxy for processes and complications not well
understood. From a certain maternal age, it is thus almost impossible to
get a test-negative result, even though most babies born by mothers that
age are healthy <span class="citation"
data-cites="schmidt_etal2007">(Schmidt et al. 2007)</span>.</p>
<p>The above formula for the posterior odds means that we can compute
the posterior probability for T21 as <span class="math display">\[
\text{Posterior probability} = \left( 1 +
\frac{1}{\operatorname{LR}_{joint}} \cdot \frac{1-\text{Prior
probability}}{\text{Prior probability}} \right)^{-1}.
\]</span> The complicated part of the analysis is now to determine the
joint likelihood ratio between the aneuploid pregnancies (i.e. T21
pregnancies) vs. euploid pregnancies (i.e. non-T21 pregnancies) as a
function of the three biomarkers. In some cases additional continuous or
binary markers indicating for example the smoking status, the visibility
of the nasal bone on the ultrasound or the ductus venosus blood flow
(expressed as pulsatility index) are used to distinguish further. For
sake of exposition we use the presence of a nasal bone on the ultrasound
as additional marker.</p>
<p>In <span class="citation" data-cites="merz_etal2011">Merz et al.
(2011)</span> and <span class="citation" data-cites="merz_etal2016">Merz
et al. (2016)</span> the likelihood ratio between aneuploid and euploid
pregnancies is broken down into individual components by assuming that
the random variables describing the observed value for the four risk
factors are independent given ploidy (i.e. the T21 status):</p>
<p><span class="math display">\[
\begin{align*}
\operatorname{LR}_{joint} &amp;= \frac{f_{\text{aneu}}(y_{\text{NT}},
y_{\beta\text{-HCG}}, y_{\text{PAPP-A}}, y_{\text{nasal
bone}})}{f_{\text{eu}}(y_{\text{NT}}, y_{\beta\text{-HCG}},
y_{\text{PAPP-A}}, y_{\text{nasal bone}})} \\
&amp;=
\left(
  \frac{f_{\text{aneu}, \text{NT}}(y_{\text{NT}})}{f_{\text{eu},
\text{NT}}(y_{\text{NT}})}
\right)
\cdot
\left(
  \frac{f_{\text{aneu},
\beta\text{-HCG}}(y_{\beta\text{-HCG}}}{f_{\text{eu},
\beta\text{-HCG}}(y_{\beta\text{-HCG}})}
\right)
\cdot
\left(
  \frac{f_{\text{aneu}, \text{PAPP-A}}(y_{\text{PAPP-A}})}{f_{\text{eu},
\text{PAPP-A}}(y_{\text{PAPP-A}})}
\right)
  \cdot
\left(
\frac{f_{\text{aneu}, \text{nasal bone}}(y_{\text{nasal
bone}})}{f_{\text{eu}, \text{nasal bone}}(y_{\text{nasal bone}})}
\right)\\
&amp;=  \operatorname{LR}_{\text{NT}} \cdot
\operatorname{LR}_{\beta\text{-HCG}} \cdot
\operatorname{LR}_{\text{PAPP-A}} \cdot \operatorname{LR}_{\text{nasal
bone}}
\end{align*}
\]</span></p>
<p>In the above, the <span class="math inline">\(f\)</span>’s denote the
respective probability density functions, which may depend on covariates
such as gestational age (at the time of measuring) or the weight of the
mother. In <a
href="/blog/figure/source/2018-06-14-prc/fts-appendix.html#appendix1">Appendix
1</a> we look in detail at how the likelihood ratio is computed for the
NT measurement. The procedure is then analogous for the β-HCG and
PAPPA-A markers - one difference is, though, that <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span> extends the
analysis to allow for these β-HCG and PAPPA serum markers to be
collected at a gestational age 2-3 weeks earlier than the NT
measurement, because the discrimination is better for these biochemistry
markers at this gestational age. Altogether, if the prior odds is given
in the form <span class="math inline">\(1:x\)</span> then <span
class="math inline">\(\operatorname{LR}_{joint}\)</span> is the factor
multiplied onto <span class="math inline">\(x\)</span> to get the
posterior odds in the form of <span class="math inline">\(1:y\)</span>.
That means that if <span
class="math inline">\(\operatorname{LR}_{joint}&gt;1\)</span> then the
marker values are such that the risk for T21 increases.</p>
<h2 id="r-implementation">R Implementation</h2>
<p>The <code>trisomy21</code> function in the <code>trisomy21risk</code>
package performs all the computations to take one from the prior odds,
covariates and biomarker values to the posterior odds.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">formalArgs</span>(trisomy21risk<span class="sc">::</span>trisomy21)</span></code></pre></div>
<pre><code>## [1] &quot;age&quot;        &quot;weight&quot;     &quot;crlbe&quot;      &quot;crl&quot;        &quot;nt&quot;         &quot;pappa&quot;     
## [7] &quot;betahCG&quot;    &quot;nasalbone&quot;  &quot;background&quot;</code></pre>
<p>In the above <code>age</code> (in years) and <code>weight</code> (in
kg) refer to the mother, <code>crl</code> is the <a
href="https://en.wikipedia.org/wiki/Crown-rump_length">crown-rump-length</a>
(measured in mm) at the time of the NT measurement and
<code>crlbe</code> is the crown-rump-length at the time of the biomarker
measurement. The <code>pappa</code> and <code>betahCG</code>
concentrations are both measured in miU/ml. Finally,
<code>background</code> is the <span class="math inline">\(x\)</span> of
the prior odds <span class="math inline">\(1:x\)</span>. The function
outputs a named list of which one element is the posterior odds (in the
form of <span class="math inline">\(1:y\)</span>).</p>
<div class="framedbox">
Philosophical note: The obtained posterior odds for the risk from the
analysis is technically not really <strong>your individual
risk</strong>, but the proportion in a population of, say, 1000
individuals with the same (measurable) covariate configuration as your
own. As Bayesians we, however, feel a certain pain towards such a
frequentist interpretation of the probability concept at the end of a
Bayesian analysis. Believing that assumptions and model choice are
subjective, we dare say that the obtained number has become <strong><a
href="https://t.co/iekXiiKGn1">your risk</a></strong> as soon as you
have to deal with the number.
</div>
<p>
<p>We can now compare our results to, e.g., the 12 randomly selected
patients listed in Tab. 5 of <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span>. Note that the
first patient has a posterior odds of 1:486, which we used as example
patient in the introduction. Results are compared in two ways:</p>
<ul>
<li>using the background risk as specified in the
<code>Background</code> column by the paper and comparing our result
(Column: <code>our_Post</code>) to the <span
class="math inline">\(y\)</span> of the posterior odds stated in the
paper (Column: <code>Post</code>).</li>
<li>Manually calculate the background risk using maternal age and
gestational age by interpolation of the Snijders table (Column:
<code>our_Background</code>) and compare our result (Column:
<code>our_Post_Back</code>) with the result of the paper (Column:
<code>Post</code>).</li>
</ul>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="do">##Load numbers from Tab. 5 of the paper (contains 12 patients)</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>tab5 <span class="ot">&lt;-</span> <span class="fu">read.csv2</span>(<span class="fu">file.path</span>(filePath,<span class="st">&quot;merzetal2015-tab5.csv&quot;</span>))</span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a><span class="do">##Compute our own results for each patient</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>tab5 <span class="sc">%&lt;&gt;%</span> rowwise <span class="sc">%&gt;%</span> <span class="fu">do</span>({</span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>  <span class="do">##Arguments for the call to the trisomy21 function</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>  args <span class="ot">&lt;-</span> <span class="fu">list</span>(<span class="at">age=</span>.<span class="sc">$</span>Age, <span class="at">weight=</span>.<span class="sc">$</span>Weight, <span class="at">crlbe=</span><span class="fu">gestage2crl</span>(.<span class="sc">$</span>CRLBE), <span class="at">crl=</span><span class="fu">gestage2crl</span>(.<span class="sc">$</span>CRL), <span class="at">nt=</span>.<span class="sc">$</span>NT, <span class="at">pappa=</span>.<span class="sc">$</span>PAPP_A, <span class="at">betahCG=</span>.<span class="sc">$</span>beta_hCG, <span class="at">nasalbone=</span><span class="st">&quot;Unknown&quot;</span>, <span class="at">background=</span>.<span class="sc">$</span>Background)</span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>  <span class="do">##Manually compute the prior odds</span></span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>  our_Background <span class="ot">&lt;-</span> <span class="fu">background_risk</span>(<span class="at">age_mother=</span>.<span class="sc">$</span>Age, <span class="at">gestage=</span>.<span class="sc">$</span>CRL<span class="sc">/</span><span class="dv">7</span>)</span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>  <span class="do">##Call the trisomy function with the two type of background odds</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>  post <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;trisomy21&quot;</span>,<span class="at">args=</span>args)</span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>  post_back <span class="ot">&lt;-</span> <span class="fu">do.call</span>(<span class="st">&quot;trisomy21&quot;</span>,<span class="at">args=</span><span class="fu">modifyList</span>(args, <span class="fu">list</span>(<span class="at">background=</span>our_Background)))</span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>  <span class="do">##Return result</span></span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>(., <span class="at">our_Post =</span> post<span class="sc">$</span>onetox, <span class="at">our_Background=</span><span class="fu">round</span>(our_Background,<span class="at">digits=</span><span class="dv">0</span>), <span class="at">our_Post_Back=</span>post_back<span class="sc">$</span>onetox)</span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>})</span></code></pre></div>
<table class="table table-striped" style="font-size: 14px; margin-left: auto; margin-right: auto;">
<caption style="font-size: initial !important;">
Table 2: Table 5 in Merz et al. (2016) annotated with our manually
computed results. Note: The gestational ages (CRL and CRLBE) are now
suddenly measured in gestation week and not mm.
</caption>
<thead>
<tr>
<th style="text-align:right;">
Age
</th>
<th style="text-align:right;">
Weight
</th>
<th style="text-align:right;">
CRLBE
</th>
<th style="text-align:right;">
CRL
</th>
<th style="text-align:right;">
NT
</th>
<th style="text-align:right;">
PAPP_A
</th>
<th style="text-align:right;">
beta_hCG
</th>
<th style="text-align:right;">
Background
</th>
<th style="text-align:right;">
Post
</th>
<th style="text-align:right;">
our_Post
</th>
<th style="text-align:right;">
our_Background
</th>
<th style="text-align:right;">
our_Post_Back
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
74.5
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
86
</td>
<td style="text-align:right;">
2.3
</td>
<td style="text-align:right;">
0.330
</td>
<td style="text-align:right;">
52.5
</td>
<td style="text-align:right;">
145
</td>
<td style="text-align:right;">
486
</td>
<td style="text-align:right;background-color: lightblue;">
494
</td>
<td style="text-align:right;background-color: lightblue;">
140
</td>
<td style="text-align:right;background-color: lightblue;">
477
</td>
</tr>
<tr>
<td style="text-align:right;">
33
</td>
<td style="text-align:right;">
65.1
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
86
</td>
<td style="text-align:right;">
2.0
</td>
<td style="text-align:right;">
0.526
</td>
<td style="text-align:right;">
124.7
</td>
<td style="text-align:right;">
349
</td>
<td style="text-align:right;">
1022
</td>
<td style="text-align:right;background-color: lightblue;">
1043
</td>
<td style="text-align:right;background-color: lightblue;">
352
</td>
<td style="text-align:right;background-color: lightblue;">
1052
</td>
</tr>
<tr>
<td style="text-align:right;">
27
</td>
<td style="text-align:right;">
48.6
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
88
</td>
<td style="text-align:right;">
1.2
</td>
<td style="text-align:right;">
0.210
</td>
<td style="text-align:right;">
71.6
</td>
<td style="text-align:right;">
876
</td>
<td style="text-align:right;">
2901
</td>
<td style="text-align:right;background-color: lightblue;">
3412
</td>
<td style="text-align:right;background-color: lightblue;">
752
</td>
<td style="text-align:right;background-color: lightblue;">
2931
</td>
</tr>
<tr>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
65.4
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
88
</td>
<td style="text-align:right;">
1.9
</td>
<td style="text-align:right;">
0.725
</td>
<td style="text-align:right;">
96.0
</td>
<td style="text-align:right;">
1021
</td>
<td style="text-align:right;">
13238
</td>
<td style="text-align:right;background-color: lightblue;">
13583
</td>
<td style="text-align:right;background-color: lightblue;">
915
</td>
<td style="text-align:right;background-color: lightblue;">
12175
</td>
</tr>
<tr>
<td style="text-align:right;">
42
</td>
<td style="text-align:right;">
72.8
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
89
</td>
<td style="text-align:right;">
2.0
</td>
<td style="text-align:right;">
0.359
</td>
<td style="text-align:right;">
77.0
</td>
<td style="text-align:right;">
45
</td>
<td style="text-align:right;">
176
</td>
<td style="text-align:right;background-color: lightblue;">
208
</td>
<td style="text-align:right;background-color: lightblue;">
35
</td>
<td style="text-align:right;background-color: lightblue;">
162
</td>
</tr>
<tr>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
69.0
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
89
</td>
<td style="text-align:right;">
1.8
</td>
<td style="text-align:right;">
0.241
</td>
<td style="text-align:right;">
47.2
</td>
<td style="text-align:right;">
147
</td>
<td style="text-align:right;">
1355
</td>
<td style="text-align:right;background-color: lightblue;">
1526
</td>
<td style="text-align:right;background-color: lightblue;">
140
</td>
<td style="text-align:right;background-color: lightblue;">
1453
</td>
</tr>
<tr>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
84.4
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
90
</td>
<td style="text-align:right;">
2.0
</td>
<td style="text-align:right;">
0.290
</td>
<td style="text-align:right;">
51.9
</td>
<td style="text-align:right;">
225
</td>
<td style="text-align:right;">
2198
</td>
<td style="text-align:right;background-color: lightblue;">
2332
</td>
<td style="text-align:right;background-color: lightblue;">
180
</td>
<td style="text-align:right;background-color: lightblue;">
1865
</td>
</tr>
<tr>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
61.6
</td>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
90
</td>
<td style="text-align:right;">
1.7
</td>
<td style="text-align:right;">
0.219
</td>
<td style="text-align:right;">
84.6
</td>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
264
</td>
<td style="text-align:right;background-color: lightblue;">
285
</td>
<td style="text-align:right;background-color: lightblue;">
47
</td>
<td style="text-align:right;background-color: lightblue;">
243
</td>
</tr>
<tr>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
57.6
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
91
</td>
<td style="text-align:right;">
1.6
</td>
<td style="text-align:right;">
0.349
</td>
<td style="text-align:right;">
75.3
</td>
<td style="text-align:right;">
677
</td>
<td style="text-align:right;">
6329
</td>
<td style="text-align:right;background-color: lightblue;">
6990
</td>
<td style="text-align:right;background-color: lightblue;">
576
</td>
<td style="text-align:right;background-color: lightblue;">
5947
</td>
</tr>
<tr>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
64.7
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
92
</td>
<td style="text-align:right;">
1.6
</td>
<td style="text-align:right;">
0.230
</td>
<td style="text-align:right;">
73.9
</td>
<td style="text-align:right;">
945
</td>
<td style="text-align:right;">
3983
</td>
<td style="text-align:right;background-color: lightblue;">
4506
</td>
<td style="text-align:right;background-color: lightblue;">
811
</td>
<td style="text-align:right;background-color: lightblue;">
3868
</td>
</tr>
<tr>
<td style="text-align:right;">
35
</td>
<td style="text-align:right;">
74.3
</td>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
92
</td>
<td style="text-align:right;">
1.5
</td>
<td style="text-align:right;">
0.399
</td>
<td style="text-align:right;">
69.6
</td>
<td style="text-align:right;">
233
</td>
<td style="text-align:right;">
5072
</td>
<td style="text-align:right;background-color: lightblue;">
5691
</td>
<td style="text-align:right;background-color: lightblue;">
229
</td>
<td style="text-align:right;background-color: lightblue;">
5593
</td>
</tr>
<tr>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
51.6
</td>
<td style="text-align:right;">
58
</td>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
2.1
</td>
<td style="text-align:right;">
0.563
</td>
<td style="text-align:right;">
142.4
</td>
<td style="text-align:right;">
338
</td>
<td style="text-align:right;">
902
</td>
<td style="text-align:right;background-color: lightblue;">
966
</td>
<td style="text-align:right;background-color: lightblue;">
287
</td>
<td style="text-align:right;background-color: lightblue;">
821
</td>
</tr>
</tbody>
</table>
<p>
<p>Altogether, we observe that the results do not fully agree, but are
in most cases of the same magnitude. Even when using the same prior odds
there are still differences, which can be explained by the use of
different polynomial <span class="math inline">\(\hat{y}(x)\)</span>
forms for NT, PAPP-A and β-HCG, because of problems reproducing the
results with the polynomials stated in <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span> - see <a
href="/blog/figure/source/2018-06-14-prc/fts-appendix.html#appendix2">Appendix
2</a>. Furthermore, the interpolation of the background risks from the
Snijders table (Column: <code>our_Background</code>) deviates:
Apparently, a different interpolation procedure is used by <span
class="citation" data-cites="merz_etal2016">Merz et al. (2016)</span> -
we could not find an explicit mentioning of how this part is done.</p>
<h2 id="shiny-app">Shiny App</h2>
<p>For better illustration we have written a small Shiny app in order to
get a better feeling of what the algorithm does, for example, by
performing a sensitivity analysis where one changes one measurement
value slightly. The App is available from:</p>
<center>
<a
href="https://hoehle.shinyapps.io/t21app/">https://hoehle.shinyapps.io/t21app/</a>
</center>
<p>
<p>and can also be ran locally, by installing the <a
href="https://github.com/hoehleatsu/trisomy21risk"><code>trisomy21risk</code></a>
package from github:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>devtools<span class="sc">::</span><span class="fu">install_github</span>(<span class="st">&quot;hoehleatsu/trisomy21risk&quot;</span>)</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>trisomy21risk<span class="sc">::</span><span class="fu">runExample</span>()</span></code></pre></div>
<h4 id="screenshots">Screenshots</h4>
Computing the T21 risk:
<center>
<img src="/blog/figure/source/2018-06-14-prc/shinyapp.png" alt="Second screenshot from the Shiny App"  width="85%">
</center>
<p>As example: the NT measurement in the ultrasound image is subject to
some measurement error - even the most skilled operator has difficulties
obtaining errors smaller than 0.2 - 0.3 mm. With the app we can easily
investigate how such measurement error influences the calculated risk.
Another aspect to investigate is the influence of the background risk
from maternal age.</p>
Visualizing the likelihood ratios:
<center>
<img src="/blog/figure/source/2018-06-14-prc/shinyapp2.png" alt="Second screenshot from the Shiny App"  width="85%">
</center>
<h2 id="discussion">Discussion</h2>
<p>With a cut-off of value of 1:150 for the posterior odds, <span
class="citation" data-cites="merz_etal2016">Merz et al. (2016)</span>
report a detection rate for T21 of 86.8% and a false positive rate of
3.42%. The 1:150 is also the threshold they suggest for further invasive
action. For posterior odds up to 1:500 the suggestion in <span
class="citation" data-cites="merz_etal2011">Merz et al. (2011)</span> is
to perform additional sonographic investigations in the second
trimester. Nowadays, one would typically perform a NIPT.</p>
<p>Was the algorithm for the computation of the risk detailed and
comprehensive as demanded by the Commission on Genetic Testing? Well,
since we -despite quite some effort- were not able to find a full
description of the FMF 2012 algorithm and, as described in <a
href="/blog/figure/source/2018-06-14-prc/fts-appendix.html#appendix2">Appendix
2</a>, were not able to reproduce the numbers in <span class="citation"
data-cites="merz_etal2016">Merz et al. (2016)</span>, we say: no. As
algorithms become more complex and -as for the FMF 2012 algorithm-
contain the combined insights of 10+ scientific papers, transparency
starts with a single easy-to-find accessible document coarsely
describing the overall algorithm. The FMF software manual, which
apparently is not accessible to the public, would have been a start.
Even better would be an open-source reference implementation! To this
date it remained unclear how the standard NT, PAPP-A and β-HCG
measurements entered the analysis and if additional components
(ethnicity? smoking status? nasal bone? heart rate? ductus flow?)
entered the risk calculation. Similarly, publishing a statistical
description of the FMF-Germany algorithm in open-access articles,
e.g. such as <span class="citation" data-cites="merz_etal2016">Merz et
al. (2016)</span>, is a step in the right direction, however, some of
the equations in the paper were either erroneous or intentionally kept
so vague, the algorithm became impossible to reconstruct. Altogether,
transparency and risk communication for this sensitive diagnostic
procedure should be improved to meet guideline standards.</p>
<p>What do you do with the obtained posterior risk once you have it?
This is indeed a very personal decision, which make the shortcomings of
statistics clear: Statisticians can annotate even the most unlikely
event with a probability of happening, but the probability is rarely
zero, so it’s your personal threshold for <em>likely enough</em>
balanced against the ethics and personal beliefs of what to do if, which
determines your actions… In this <span
class="math inline">\(n=1\)</span> context the reported risk might
merely be a hypothetical number - at most its rough magnitude is of
interest. However, from a system perspective <span
class="math inline">\(n\)</span> is much larger, so everything possible
should be done such that the probabilistic forecast is both calibrated
and sharp and that the recipient(s) knows what this number entails.</p>
<h4 id="postscript">Postscript</h4>
<p>Further information about first trimester screening can be obtained
from the <a href="https://fetalmedicine.org/">Fetal Medicine
Foundation</a>. In particular their <a
href="http://www.fetalmedicine.com/fmf/FMF-English.pdf">book</a> about
the 11-13 weeks scan (available in several languages) is very helpful
<span class="citation" data-cites="FMF2004">(Nicolaides 2004)</span>. As
mentioned the FMF-UK offers a web-based <a
href="https://fetalmedicine.org/research/assess/trisomies">trisomies
risk assessment calculator</a>. Similarly, the <a
href="https://www.fmf-deutschland.info/en/fmf-deutschland/">FMF-Germany</a>
has their own website with information - apparently it is now also
possible to obtain a demo version of their risk calculation software <a
href="https://www.fmf-deutschland.info/en/software/">PRC 3.0</a>.</p>
<h2 id="update-2018-07-24">Update 2018-07-24</h2>
<p>Directly after publishing the post I contacted the German <a
href="https://www.rki.de/EN/Content/Institute/Committees/GEKO/GEKO_content.html">Commission
on Genetic Testing (GEKO)</a> to make them aware of the post and my
troubles to determine what the algorithm actually does, what information
enters and the lack of will of the FMF-Germany authors to communicate
certain details of the algorithm. As a consequence, I asked which steps
the commission performs to ensure and track the realisation of the
guideline’s demands for “<em>publication of the entire algorithm with
detailed software specification</em>”. In their email reply from
2018-07-16 they answer<a href="#fn1" class="footnote-ref" id="fnref1"
role="doc-noteref"><sup>1</sup></a>: “<em>… The area of responsibility
of the GEKO is clearly regulated by legal requirements. The
implementation of the Gene Diagnostic Act (GenDG) or the examination of
the guideline’s requirements are not tasks of the GEKO. The
interpretation of the regulatory content of the Gene Diagnostic Act as
well as the enforcement of the requirements of the GEKO guidelines are
dealt with by the appropriate authorities of the Länder.</em>” As a
consequence of this, their recommendation was to address my question to
the equivalent of the Ministry of Health in my federal state (Berlin). I
did this on 2018-07-16 (awaiting reply), but would be surprised, if the
Federal State of Berlin has the resources and expertise to look into
software transparency of national and internationally operating software
vendors in this field. I doubt this is different for any of the 15 other
federal states in Germany. I might be wrong and in for a surprise, but
as a citizen and scientist it seems like the guideline’s demand for a
detailed software specification in order to make it transparent and
reproducible how the 1:x is calculated is nothing else than wishful
thinking…</p>
<h2 id="update-2019-03-29">Update 2019-03-29</h2>
<p>After several inquiries about what is done to ensure the algorithmic
transparency of the prenatal diagnostic procedures remained unanswered
by Berlin’s <a
href="https://www.berlin.de/sen/gpg/"><em>Senatsverwaltung für
Gesundheit, Pflege und Gleichstellung</em></a>, I decided to formulate a
<a
href="http://gesetze.berlin.de/jportal/?quelle=jlink&amp;query=InfFrG+BE&amp;psml=bsbeprod.psml&amp;max=true&amp;aiz=true">freedom
of information act</a> request, which ensures that the public authority
has to answer, typically within ~4 weeks. A very good interface to
formulate such requests in Germany is <a
href="https://fragdenstaat.de">fragdenstaat.de</a>. The <a
href="https://fragdenstaat.de/anfrage/vorgeburtliche-risikoabklarung/">subsequent
answer</a> (in German) shows that no measures are performed in Berlin to
ensure the requested algorithmic transparency. My conclusion: The
current setup of a) recommendations being in the hand of the GEKO while
b) compliance assurance being in the hand of 16 federal states results
in the software providers not being under any pressure to provide
algorithmic transparency. This is rather frustrating for an ethical
sensitive issue such as prenatal screening.</p>
<h2 id="appendix">Appendix</h2>
<p>The mathematical details of the post and our troubles reproducing the
results of <span class="citation" data-cites="merz_etal2016">Merz et al.
(2016)</span> are described in two separate <a
href="/blog/figure/source/2018-06-14-prc/fts-appendix.html">Appendixes</a>,</p>
<h2 class="unnumbered" id="literature">Literature</h2>
<div id="refs" class="references csl-bib-body hanging-indent"
role="list">
<div id="ref-kagan_etal2008a" class="csl-entry" role="listitem">
Kagan, K. O., D. Wright, A. Baker, D. Sahota, and K. H. Nicolaides.
2008. <span>“<span class="nocase"><span>S</span>creening for trisomy 21
by maternal age, fetal nuchal translucency thickness, free beta-human
chorionic gonadotropin and pregnancy-associated plasma
protein-<span>A</span></span>.”</span> <em>Ultrasound Obstet
Gynecol</em> 31 (6): 618–24.
</div>
<div id="ref-kagan_etal2008b" class="csl-entry" role="listitem">
Kagan, K. O., D. Wright, C. Valencia, N. Maiz, and K. H. Nicolaides.
2008. <span>“<span class="nocase"><span>S</span>creening for trisomies
21, 18 and 13 by maternal age, fetal nuchal translucency, fetal heart
rate, free beta-h<span>C</span><span>G</span> and pregnancy-associated
plasma protein-<span>A</span></span>.”</span> <em>Hum. Reprod.</em> 23
(9): 1968–75.
</div>
<div id="ref-merz_etal2008" class="csl-entry" role="listitem">
Merz, E., C. Thode, A. Alkier, B. J. Hackelöer, M. Hansmann, G. Huesgen,
P. Kozlowski, M. Pruggmaier, and S. Wellek. 2008. <span>“A New Approach
to Calculating the Risk of Chromoso- Mal Abnormalities with
First-Trimester Screening Data.”</span> <em>Ultraschall in Der Medizin /
European Journal of Ultrasound</em> 29: 639–45. <a
href="https://doi.org/10.1055/s-2008-1027958">https://doi.org/10.1055/s-2008-1027958</a>.
</div>
<div id="ref-merz_etal2011" class="csl-entry" role="listitem">
Merz, E., C. Thode, B. Eiben, R. Faber, B. J. Hackelöer, G. Huesgen, M.
Pruggmaier, and S. Wellek. 2011. <span>“Individualized Correction for
Maternal Weight in Calculating the Risk of Chromosomal Abnormalities
with First-Trimester Screening Data.”</span> <em>Ultraschall in Der
Medizin/European Journal of Ultrasound</em> 32: 33–39.
https://doi.org/<a
href="http://dx.doi.org/10.1055/">http://dx.doi.org/10.1055/</a>.
</div>
<div id="ref-merz_etal2016" class="csl-entry" role="listitem">
Merz, E., C. Thode, B. Eiben, and S. Wellek. 2016. <span>“Prenatal Risk
Calculation (<span>PRC</span>) 3.0: An Extended DoE-Based
First-Trimester Screening Algorithm Allowing for Early Blood
Sampling.”</span> <em>Ultrasound International Open</em> 2: E19–26. <a
href="https://doi.org/10.1055/s-0035-1569403">https://doi.org/10.1055/s-0035-1569403</a>.
</div>
<div id="ref-FMF2004" class="csl-entry" role="listitem">
Nicolaides, K. H. 2004. <em>The 11–13<span
class="math inline">\(^{+6}\)</span> Weeks Scan</em>. Fetal Medicine
Foundation,. <a
href="https://fetalmedicine.com/synced/fmf/FMF-English.pdf">https://fetalmedicine.com/synced/fmf/FMF-English.pdf</a>.
</div>
<div id="ref-schmidt_etal2007" class="csl-entry" role="listitem">
Schmidt, P., J. Rom, H. Maul, B. Vaske, P. Hillemanns, and A. Scharf.
2007. <span>“<span class="nocase"><span>A</span>dvanced first trimester
screening (<span>A</span><span>F</span><span>S</span>): an improved test
strategy for the individual risk assessment of fetal aneuploidies and
malformations</span>.”</span> <em>Arch. Gynecol. Obstet.</em> 276 (2):
159–66.
</div>
<div id="ref-snijders_etal1999" class="csl-entry" role="listitem">
Snijders, R. J., K. Sundberg, W. Holzgreve, G. Henry, and K. H.
Nicolaides. 1999. <span>“<span class="nocase"><span>M</span>aternal age-
and gestation-specific risk for trisomy 21</span>.”</span>
<em>Ultrasound Obstet Gynecol</em> 13 (3): 167–70.
</div>
<div id="ref-wright_etal2008" class="csl-entry" role="listitem">
Wright, D., K. O. Kagan, F. S. Molina, A. Gazzoni, and K. H. Nicolaides.
2008. <span>“<span class="nocase"><span>A</span> mixture model of nuchal
translucency thickness in screening for chromosomal
defects</span>.”</span> <em>Ultrasound Obstet Gynecol</em> 31 (4):
376–83.
</div>
</div>
<section id="footnotes" class="footnotes footnotes-end-of-document"
role="doc-endnotes">
<hr />
<ol>
<li id="fn1"><p>My email and their answer was in German - the quoted
part of the answer is translated to English by me.<a href="#fnref1"
class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>

  </div>

</article>

	

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">Theory meets practice...</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col footer-col-1">
        <ul class="contact-list">
          <li>Theory meets practice...</li>
          <li><a href="https://math-inf.uni-greifswald.de/en/michael-hoehle/">Michael Höhle</a></li>
        </ul>
      </div>

      <div class="footer-col footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/mhoehle"><span class="icon icon--github"><svg viewBox="0 0 16 16"><path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/></svg>
</span><span class="username">mhoehle</span></a>

          </li>
          

          
          <li>
            <a href="https://twitter.com/m_hoehle"><span class="icon icon--twitter"><svg viewBox="0 0 16 16"><path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/></svg>
</span><span class="username">m_hoehle</span></a>

          </li>
          
        </ul>
      </div>

      <div class="footer-col footer-col-3">
        <p>A blog about statistics in theory and practice. Not always serious, not always flawless, but definitely a statistically flavoured bean.
</p>
      </div>
    </div>

  </div>

</footer>


  </body>

</html>
